### PR TITLE
Customize the environment for CDMS

### PR DESCRIPTION
**Pull request by zonca**
Implementing https://github.com/det-lab/jupyterhub-deploy-kubernetes-jetstream/issues/3

### CODE REVIEW

File: config_standard_storage.yaml [hunk 1]
```diff
@@ -18,15 +18,20 @@ proxy:
...
+    limit: 10
   storage:
     type: dynamic
-    capacity: 1Gi
+    capacity: 5Gi
     dynamic:
       storageClass: standard
+  image:
+    name: supercdms/cdms-jupyterlab
+    tag: 1.8b
```
Comment (zonca) [inline config_standard_storage.yaml]: in you files you had `0.7b` which doesn't exist at: https://hub.docker.com/r/supercdms/cdms-jupyterlab/tags
I am trying with `1.8b`

Comment (zonca) [reply config_standard_storage.yaml]: @pibion

**Comment by pibion** [comment]
@zonca do you recommend deleting the branch after merging?

**Comment by zonca** [comment]
@pibion please leave this open, we can close it once the work is done. It is useful for me to track what is in progress. I will take care of reopening this and undo the changes in master.

**Pull request merged**